src/github_runner_image_builder/templates/cloud-init.sh.j2

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+function configure_proxy() {
+    local proxy="$1"
+    if [[ -z "$proxy" ]]; then
+        return
+    fi
+    echo "Installing aproxy"
+    /usr/bin/sudo snap install aproxy --edge;
```

Comment:
Off topic. Why edge?

Reply:
The aproxy doesn't have a stable release yet. @weiiwang01 Could this now be promoted?